src/prefect/__init__.py

```diff
@@ -23,8 +23,8 @@
 if sys.version_info < (3, 8):
     warnings.warn(
         (
-            "Prefect will drop support for Python 3.7 when it reaches end-of-life on 27"
-            " Jun 2023. To use new versions of Prefect after that date, you will need"
+            "Prefect dropped support for Python 3.7 when it reached end-of-life on 27"
+            " Jun 2023. To use new versions of Prefect, you will need"
```

Comment:
let's say July 6

Reply:
just removing the data so it's not a confusing issue.